pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -0,0 +1,38 @@
+package harness_runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prow"
+)
+
+var _ = ginkgo.Describe("[Suite: harnessess] Test Harness", func() {
+	defer ginkgo.GinkgoRecover()
+	h := helper.New()
+
+	TimeoutInSeconds := float64(viper.GetFloat64(config.Tests.PollingTimeout))
```

Comment:
nit: is there a reason we use float64 for polling timeout? It is commonly used all over osde2e but then below it is just converted to an int anyway

Reply:
No actual reason here, there isn't any standard across the tests. 
The reason it turns into an int is just for the helper RunTests function.